vueapp/components/Videos/VideoCard.vue

```diff
@@ -241,6 +251,16 @@ export default {
                 >= 0 ? true : false;
         },
 
+        getAccessText() {
```

Comment:
Irgendwie müsste das noch mit dem Huma-readable Text hier kombiniert werden:
https://github.com/elan-ev/studip-opencast-plugin/blob/master/vueapp/components/Videos/Actions/VideoAccess.vue#L210

Könnte man das evtl. als "Filter" in `vueapp/common/permname.filter.js` einbauen?